docs/src/integrations/profiling/new-relic/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "New Relic"
+description: |
+  Platform.sh supports [New Relic APM](https://newrelic.com/products/application-monitoring) for profiling your applications.
+---
+
+{{< description >}}
+
+## On a Dedicated cluster
+
+Open a support ticket and let us know what your license key is.  Our support team will install it and let you know when it is complete.
+
+## On a Grid plan, configure your project
```

Comment:
This seems to just stop in the middle here?

Reply:
It then displays the list of the supported languages but hugo does that automatically. Looks weird, I agree. 🤷

Reply:
Oh ew. I really don't like the default giant-button design there. :smile: It could still use some additional intro text, like "the setup varies slightly by language, select the appropriate language from the list below" or something.